Client/src/Components/Avatar/index.jsx

```diff
@@ -15,29 +16,48 @@ import PropTypes from "prop-types";
  * <Avatar src="assets/img" first="Alex" last="Holliday" small />
  */
 
-const Avatar = ({ src, firstName, lastName, small, sx }) => {
-  const borderColor = "#F0F2F4";
-  const smallStyle = small ? { width: 32, height: 32 } : {};
+const Avatar = ({ src, small, sx }) => {
+  const { user } = useSelector((state) => state.auth);
+
+  const style = small ? { width: 25, height: 25 } : { width: 64, height: 64 };
+  const border = small ? 1 : 3;
+
+  const [image, setImage] = useState();
+  useEffect(() => {
+    if (user.avatarImage) {
+      setImage(`data:image/png;base64,${user.avatarImage}`);
+    }
+  }, [user?.avatarImage]);
+
   return (
     <MuiAvatar
-      alt={`${firstName} ${lastName}`}
-      src={src}
+      alt={`${user?.firstname} ${user?.lastname}`}
+      children={`${user.firstname?.charAt(0)}${user.lastname?.charAt(0)}`}
```

Comment:
React doesn't like this:

`Do not pass children as props. Instead, nest children between the opening and closing tags.eslint[react/no-children-prop](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/no-children-prop.md)`

Any reason they can't be passed regularly inbetween the tags?

Reply:
No reason in particular, I was following the @mui docs. Will pass the initials inside the tags.